floris/tools/floris_interface.py

```diff
@@ -558,12 +558,20 @@ def check_wind_condition_for_viz(self, wd=None, ws=None):
         if len(ws) > 1 or len(ws) < 1:
             raise ValueError("Wind speed input must be of length 1 for visualization. Current length is {}.".format(len(ws)))
 
+    def check_calc_wake_run(self, func_name):
```

Comment:
Maybe make this private, `_check_calc_wake_run`?

Reply:
Changed